src/styles/controls.plugin.scss

```diff
@@ -0,0 +1,51 @@
+.wikia-jw-small-player-controls {
+	.wikia-jw-small-player-controls-plugin {
+		align-items: center;
+		display: flex;
+		fill: white;
+		height: 100%;
+		justify-content: center;
+		left: 0;
+		opacity: 0;
+		position: absolute;
+		top: 0;
+		transition: all .1s linear .1s;
```

Comment:
As there is no animation for the scrubber bar I would get rid of this one as well :)

Reply:
There is a transition for the bar. ;)